templates/people_admin/people_matcher.html

```diff
@@ -14,16 +14,18 @@
 				<div class="cell medium-5">
 					<h1 class="heading--large">People Matcher</h1>
 					<h2>{{ session }}</h2>
+                    <h4>Unmatched Total: {{ unmatched_total }}</h4>
 				</div>
 				<div class="cell medium-5">
 					</br>
-					<select id="session" name="session" onchange="window.location.href = '/dashboard/people/{{ state }}_matcher/' + this.value">
+					<select id="session" name="session"
+                            onchange="window.location.href = '/admin/people/{{ state }}/matcher/' + this.value">
```

Comment:
Could not figure out how to make this work with the url reference and still work.